Tropos/Models/Temperature.swift

```diff
@@ -13,7 +13,7 @@ func -(lhs: Temperature, rhs: Temperature) -> Temperature {
     return Temperature(fahrenheitValue: lhs.fahrenheitValue - rhs.fahrenheitValue)
 }
 
-@objc class Temperature: NSObject, Printable {
+@objc class Temperature: NSObject {
```

Comment:
If I remember correctly, I think Xcode was now complaining that it was redundant since NSObject already does it. We still have the implementation https://github.com/thoughtbot/Tropos/blob/jc-swift-upgrade/Tropos/Models/Temperature.swift#L42-L45